packages/devtools-scripts/src/main.js

```diff
@@ -0,0 +1,218 @@
+import { aop, createHook, hookName, createCallTrap } from 'to-aop';
+import uid from 'easy-uid';
+import DevToolManager from './DevToolManager';
+
+const ImaMainModules = [
+  'onLoad',
+  'getInitialImaConfigFunctions',
+  'getNamespace',
+  'getInitialPluginConfig',
+  'createImaApp',
+  'getClientBootConfig',
+  'bootClientApp',
+  'routeClientApp',
+  'hotReloadClientApp',
```

Comment:
The key hotReloadClientApp was removed from IMA@17

Reply:
Fixed.